### PR TITLE
modules/darwin/telegraf: add swap

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: modules/darwin/common/telegraf.nix [hunk 1]
```diff
@@ -7,6 +7,7 @@
         prometheus.metric_version = 2;
         system = { };
         mem = { };
+        swap = { };
```
Comment (Mic92) [inline modules/darwin/common/telegraf.nix:10]: Do you want me to add checks for swap?

Comment (Mic92) [reply modules/darwin/common/telegraf.nix:10]: Ah. I got one.

File: modules/darwin/common/telegraf.nix [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
         disk.tagdrop = {
           fstype = [ "tmpfs" "ramfs" "devtmpfs" "devfs" "iso9660" "overlay" "aufs" "squashfs" ];
```
Comment (Mic92) [inline modules/darwin/common/telegraf.nix:12]: Pretty sure most of them don't exist on macos i.e. overlay aufs squashfs

File: modules/darwin/common/telegraf.nix [hunk 3]
```diff
@@ -7,6 +7,7 @@
...
           device = [ "rpc_pipefs" "lxcfs" "nsfs" "borgfs" ];
```
Comment (Mic92) [inline modules/darwin/common/telegraf.nix:13]: Maybe borgfs exists? The rest probably doesn't.

File: modules/darwin/common/telegraf.nix (hunk 2 above)
Comment (zowoq) [reply modules/darwin/common/telegraf.nix:12]: Yes, I just copied `disk.tagdrop` from the srvos module so it's easier to keep them in sync.

**Comment by zowoq** [comment]
Deployed to darwin hosts.

**Pull request merged**